etc/actionlint.yml

```diff
@@ -0,0 +1,5 @@
+---
+self-hosted-runner:
```

Comment:
We use gh hosted runners.  According to [this](https://github.com/rhysd/actionlint/blob/94c4f0119fdc7f776c7a472a981121cafd17a3cd/docs/config.md#configuration-file) shouldn't this just be empty then?